onnxruntime/core/providers/openvino/openvino_provider_factory.cc

```diff
@@ -74,13 +76,17 @@ ORT_API_STATUS_IMPL(OrtSessionOptionsAppendExecutionProviderEx_OpenVINOEP,
       }
     } else if(key == "device_id") {
       device_id = value;
+    } else if (key == "num_of_threads") {
+      size_t n_t = std::stoi(value);
+      num_of_threads = n_t;
```

Comment:
add validation to make sure it's >0.

Reply:
> add validation to make sure it's >0.

@HectorSVC This validation will be anyway taken care at the later stage in openvino_execution_provider.cc, I have added the check there.
https://github.com/microsoft/onnxruntime/pull/5568/files#diff-759bcefaa09c60fd71eca35edf2de70ded71c206a14dfb314f549d7c7770ebaf

Anyways, just to be double sure, I have added a check here as well.